### PR TITLE
 [feature-wip](unique-key-merge-on-write) unique key table with MOW supports sequence column

### PR DESCRIPTION
**Pull request by liaoxin01**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhannngchen** [comment]
Nice work

File: be/src/olap/rowset/segment_v2/segment.cpp [hunk 1]
```diff
@@ -245,19 +245,59 @@ Status Segment::new_bitmap_index_iterator(const TabletColumn& tablet_column,
...
+        std::unique_ptr<ColumnVectorBatch> cvb;
+        RETURN_IF_ERROR(ColumnVectorBatch::create(num_to_read, false, _pk_index_reader->type_info(),
+                                                  nullptr, &cvb));
+        ColumnBlock block(cvb.get(), &pool);
+        ColumnBlockView column_block_view(&block);
+        size_t num_read = num_to_read;
+        RETURN_IF_ERROR(index_iterator->next_batch(&num_read, &column_block_view));
+        DCHECK(num_to_read == num_read);
+
+        const Slice* previous_key = reinterpret_cast<const Slice*>(cvb->cell_ptr(0));
```
[the remark this answers is not in the document]
Comment (liaoxin01) [reply be/src/olap/rowset/segment_v2/segment.cpp]: fixed

File: be/test/olap/primary_key_index_test.cpp [hunk 2]
```diff
@@ -56,7 +56,7 @@ TEST_F(PrimaryKeyIndexTest, builder) {
     auto fs = io::global_local_filesystem();
     EXPECT_TRUE(fs->create_file(filename, &file_writer).ok());
 
-    PrimaryKeyIndexBuilder builder(file_writer.get());
+    PrimaryKeyIndexBuilder builder(file_writer.get(), 0);
```
[the remark this answers is not in the document]
Comment (liaoxin01) [reply be/test/olap/primary_key_index_test.cpp:59]: done

**Review by zhannngchen — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**